get-invitees.js

```diff
@@ -1,52 +0,0 @@
-'use strict';
```

Comment:
No longer need this file because we are going to use `getLatestResponses` instead.